lib/rubygems/remote_fetcher.rb

```diff
@@ -330,7 +330,10 @@ def cache_update_path uri, path = nil, update = true
     if update and path
       begin
         open(path, 'wb') do |io|
-          io.flock(File::LOCK_EX)
+          begin
+            io.flock(File::LOCK_EX)
+          rescue Errno::ENOTSUP
```

Comment:
Also, looking at this again, is *just not locking the file* a good response to flock failing?

It kind of defeats the point of locking it...